crates/astria-core/src/sequencer/v1/test_utils.rs

```diff
@@ -27,16 +27,33 @@ pub fn make_cometbft_block() -> tendermint::Block {
     .make()
 }
 
+#[derive(Default)]
+pub struct UnixTimeStamp {
+    pub secs: i64,
+    pub nanos: u32,
+}
+
+impl From<(i64, u32)> for UnixTimeStamp {
+    fn from(val: (i64, u32)) -> Self {
+        Self {
+            secs: val.0,
+            nanos: val.1,
+        }
+    }
+}
+
 /// Allows configuring a Comet BFT block setting the height, signing key and
 /// proposer address.
 ///
 /// If the proposer address is not set it will be generated from the signing key.
 #[derive(Default)]
 pub struct ConfigureCometBftBlock {
+    pub chain_id: Option<String>,
     pub height: u32,
     pub proposer_address: Option<tendermint::account::Id>,
     pub signing_key: Option<ed25519_consensus::SigningKey>,
     pub rollup_transactions: Vec<(RollupId, Vec<u8>)>,
+    pub unix_timestamp: UnixTimeStamp,
```

Comment:
This replaces the call to `Time::now()` - having non-deterministic bits in tests is bad because that changes the block hashes.